### ISSUE TITLE
Atom styled-components comment reformated

### ISSUE THREAD

**Issue by k15a** [prettier/prettier#1589]
Hey,

Thank you for this great tool ❤️

When you comment out some css of a styled-component in Atom it will comment out the line the following way:

```js
const Comp = styled.div`
    ${''/* border-top: 4px; */}
`
```

Prettier then formats this to:

```js
const Comp = styled.div`
    ${/* border-top: 4px; */ ''}
`
```

which is a bit annoying because you can't uncomment it with the same shortcut.
Would be great if this could be fixed if it's an easy fix.

**Comment by vjeux** [comment]
If we fix it, we would add a space after '', can you tell me if using the comment key uncomments the following block of code?

```js
const Comp = styled.div`
    ${'' /* border-top: 4px; */}
`
```

If it does, then we should fix it (I don't expect it to be super hard), but if it doesn't, then I'm sorry but we won't remove that space just to please atom integration.

**Comment by k15a** [comment]
Unfortunately it doesn't. I totally agree with you that it's not worth changing just for Atom.

**Comment by vjeux** [comment]
Do you mind raising this on the atom plugin repo? Maybe they can fix it there and we can fix it in prettier as well. This way we can actually use this very useful shortcut :)

**Comment by k15a** [comment]
`language-babel` just pushed a commit which adds an extra space after the empty string. I will try my best to send a PR.

**Comment by vjeux** [comment]
Nice!

**Comment by k15a** [comment]
I've tested a bit around and noticed that this is only working with single quotes option enabled. Do you think it's still worth pushing the update?

**Comment by vjeux** [comment]
Yeah, most people are actually using single quotes when using prettier.

**Comment by MrLoh** [comment]
I'm really confused as to what the state of this is, but it is crucial to get this fixed for using babel and prettier together, which are both such amazing projects. What can be done to move this forward?
